tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -214,6 +217,87 @@ def test_fsx_lustre_backup(region, pcluster_config_reader, clusters_factory, os,
     _test_delete_manual_backup(remote_command_executor, manual_backup, region)
 
 
+@pytest.mark.dimensions("us-west-2", "c5.xlarge", "alinux2", "slurm")
+@pytest.mark.usefixtures("instance")
+def test_existing_fsx(
+    region, existing_fsx, pcluster_config_reader, clusters_factory, os, scheduler, vpc_stack, test_datadir
+):
+    """
+    Test existing Fsx file system
+
+    Check fsx_fs_id provided in config file can be mounted correctly
+    """
+    mount_dir = "/fsx_mount_dir"
+    bucket_name = existing_fsx[1]
+    cluster_config = pcluster_config_reader(
+        bucket_name=bucket_name, mount_dir=mount_dir, fsx_fs_id=existing_fsx[0].cfn_resources["existingfsx"]
+    )
+    cluster = clusters_factory(cluster_config)
+
+    _test_fsx_lustre(cluster, region, scheduler, os, mount_dir, bucket_name, storage_type=None, auto_import_policy=None)
+
+
+@pytest.fixture()
+def existing_fsx(vpc_stack, cfn_stacks_factory, request, region, key_name, s3_bucket_factory, test_datadir):
+    # Create s3 bucket and upload test_file
+    bucket_name = s3_bucket_factory()
+    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
+    bucket.upload_file(str(test_datadir / "s3_test_file"), "s3_test_file")
+
+    # FSx stack
+    fsx_template = Template()
+    fsx_template.set_version()
+    fsx_template.set_description("Create FSx stack for testing existing FSx")
+
+    # Create security group. If using an existing file system
+    # It must be associated to a security group that allows inbound TCP traffic to port 988
+    existing_fsx_sg = ec2.SecurityGroup(
+        "ExistingFSxSecurityGroup",
+        GroupDescription="SecurityGroup for testing existing FSx",
+        SecurityGroupIngress=[
+            ec2.SecurityGroupRule(
+                IpProtocol="tcp",
+                FromPort="988",
+                ToPort="988",
+                CidrIp="0.0.0.0/0",
+            ),
+        ],
+        VpcId=vpc_stack.cfn_outputs["VpcId"],
+    )
+
+    fsx_filesystem = FileSystem(
+        title="existingfsx",
+        FileSystemType="LUSTRE",
+        StorageCapacity=1200,
+        SecurityGroupIds=[Ref(existing_fsx_sg)],
+        SubnetIds=[vpc_stack.cfn_outputs["PublicSubnetId"]],
+        LustreConfiguration=LustreConfiguration(
+            title="lustreConfiguration",
+            ImportPath="s3://{0}".format(bucket_name),
+            ExportPath="s3://{0}/export_dir".format(bucket_name),
+        ),
+        DependsOn=["ExistingFSxSecurityGroup"],
```

Comment:
minor: this `DependsOn` is optional, because there is `Ref(existing_fsx_sg)`. I am fine with keeping it.
Here is a paragraph from the document of cloudFormation

> Dependent stacks also have implicit dependencies in the form of target properties !Ref and !GetAtt. For example, if the properties of resource A use a !Ref to resource B, the following rule apply:
>     Resource B is created before resource A.
>     Resource A is deleted before resource B.
>     Resource B is updated before resource A.